Search.py

```diff
@@ -220,10 +220,10 @@ def progression_locations(self):
     #
     # Win condition can be a string that gets mapped to a function(state_list) here
     # or just a function(state_list)
```

Comment:
This comment seems out of date as well.
```suggestion
    # predicate must be a function (state) -> bool, that will be applied to all states
```